deliverables/ObligatoriskOppgave2.md

```diff
@@ -0,0 +1,157 @@
+
+
+## Retrospective and improvements
+
+In retrospect regarding the first two assignments, things are being handled in a timely and professional manner. Even though we are faced with different time schedules in terms of lectures and extra curricular activities, we counter the increased difficulty of scheduling by having an extra meeting every tuesday if necessary, and try to plan have this meeting slot open. Luckily the team is somewhat flexible when it comes to schedules.
+
+
+
+
+Throughout this second assignment, some of the roles were swapped, because Tor Kristian and Tobias were away, they were employed to the secretary role and handled the written prompts for this assignment. This was to make things easier, to combat the fact that it would be hard for them to partake in pair-programming via an online call, as they were with family and unsure of when they could work. Additionally, when Tor Kristian and Tobias had any questions, they asked via Slack.
+
+Prior to the release of the second assignment, we as a group already had made plans of what to do next. We believe that staying ahead a bit will prove to be a smart investment of time and benefit us in future assignments. If we continue development between assignments, the workload when the actual assignment is released is diminished. Even if what we develop isn’t a part of the next assignment, it will still be useful for the final product.
+
+Although the team were flexible during this assignment, Tor Kristian and Tobias will have to make up for their absence with developing in the future, as having them do the written assignments would be unfair to the rest of the team.
+
+
+We should also improve at making sure everyone is working and committing their work to gitHub as it might look like someone is not contributing or contributing less to the project. It's a group project, and everyone should contribute somewhat equally. 
+
+
+## Commits 
+
+During this assignment, Tor Kristian and Tobias will have less commits because of the teams decision to give them the secretary role and to do the written assignment together. 
+
+
+ # Task 2 - Requirements 
+
+## User Stories 
+
+We generally went light on user stories for this release as most of the implementation is back-end related and as such has very little visibility to any users.
+As a user I expect the game does not allow me to do illegal moves
+The game should handle collisions between players or a user moving outside the bounds of the game without crashing or throwing errors.
+Criteria: Once the game has initialized and we have created a set amount of players, attempting to move a robot of the grid itself should not work. Neither should players be able to move onto tiles already occupied unless they have cards that can for example push other robots away.
```

Comment:
We generally went light on user stories for this release as most of the implementation is back-end related and as such has very little visibility to any users.
* As a user I expect the game does not allow me to do illegal moves
  * The game should handle collisions between players or a user moving outside the bounds of the game without crashing or throwing errors.
  * **Criteria:** Once the game has initialized and we have created a set amount of players, attempting to move a robot of the grid itself should not work. Neither should players be able to move onto tiles already occupied unless they have cards that can for example push other robots away.